internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -311,3 +320,35 @@ func setCPUQuota(cpuMountPoint, parentDir string, c *oci.Container, enable bool)
 
 	return nil
 }
+
+// RestoreIrqBalanceConfig restores irqbalance service with original banned cpu mask settings
+func RestoreIrqBalanceConfig(irqBalanceConfigFile, irqBannedCPUConfigFile string) error {
```

Comment:
Thanks for adding this restore function. Can we check if the irqbalance service is enabled before restart it?

Reply:
sure, done.

Reply:
looks good to me